src/sentry/features/__init__.py

```diff
@@ -101,6 +101,7 @@
 default_manager.add("organizations:metrics", OrganizationFeature, True)
 default_manager.add("organizations:metrics-extraction", OrganizationFeature)
 default_manager.add("organizations:minute-resolution-sessions", OrganizationFeature)
+default_manager.add("organizations:mobile-vitals", OrganizationFeature, True)
```

Comment:
Did you also mean to add this to `server.py`? I guess it would just default to `False` if it's not there and it isn't registered in Flagr, right?

Reply:
yeah i forgot to push the file

Reply:
Is it supposed to be `true` or `false`? cause right now it's `true` which i think it means its default on for everyone